web/src/lib/types/types.ts

```diff
@@ -197,6 +197,17 @@ export interface Opportunity {
   status?: string;
 }
 
+export interface QuickAction {
+  id: string;
+  name: string;
+  urlSlug: string;
+  filename: string | undefined;
+  content: string;
+  ctaLink: string | undefined;
+  ctaText: string | undefined;
+  form: string | undefined;
```

Comment:
We don't actually include that on any code type definitions because it's just something the CMS stores in the markdown - we never want to be writing any code touching it